src/js/user-profile/helpers.jsx

```diff
@@ -1,18 +1,31 @@
 import Raven from 'raven-js';
 
-export const formTitles = {
-  '21P-527EZ': 'Veterans pension benefits (21P-527EZ)',
-  '21P-530': 'burial benefits (21P-530)',
-  '1010ez': 'health care (10-10EZ)',
-  '22-1990': 'education benefits (22-1990)',
-  '22-1990E': 'education benefits (22-1990E)',
-  '22-1990N': 'education benefits (22-1990N)',
-  '22-1995': 'education benefits (22-1995)',
-  '22-5490': 'education benefits (22-5490)',
-  '22-5495': 'education benefits (22-5495)',
+export const formBenefits = {
```

Comment:
We could make these three exported objects Maps, I think.

Reply:
Excellent idea, changed.